package/yast2-installation-control.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Wed Apr 17 15:19:17 UTC 2024 - Stefan Schubert <schubi@suse.com>
+
+- Added encryption_method and encryption_pbkdf in the product
+  description file (section partitioning/proposal) in order to
+  encrypt the generated partitions.
+- 5.0.1  
```

Comment:
I'm not sure if this changelog entry will pass the army of OBS bots. I suggest to add a bugzilla reference. For example https://bugzilla.suse.com/show_bug.cgi?id=1185291

Reply:
Yes, I have adapted it.